app/src/main/res/values/dimens.xml

```diff
@@ -83,4 +83,12 @@
     <dimen name="bottom_prompt_chip_margin_bottom">28dp</dimen>
     <dimen name="bottom_prompt_chip_corner_radius">8dp</dimen>
 
+<!--    Image editing labels-->
+    <dimen name="btnAddImage_margin_bottom">80dp</dimen>
+    <dimen name="btnChooseImage_width">120dp</dimen>
+    <dimen name="label_margin_end">30dp</dimen>
+    <dimen name="btnUnselectImage_margin_bottom">76dp</dimen>
+    <dimen name="btnEditImage_margin_end">30dp</dimen>
+    <dimen name="btnEditImage_width">100dp</dimen>
+
```

Comment:
Instead of using this resources, could we use like an already defined one (the one from material)? There is no much added value in symmetry here and hard-coding these values just for a screen could be bad.

Reply:
Yes you are right.

Reply:
In this I was not thinking about other screen sizes and restricted myself to the emulator screen only, and hard coded the dimensions, we need to write a generic code to support all screen sizes and for future improvements / fixes if needed.

Thankyou for helping me out.